src/components/RadioButtonGroup/index.js

```diff
@@ -45,8 +66,49 @@ class RadioButtonGroup extends Component {
         return undefined;
     }
 
+    getCheckedOptionRef() {
+        const { value, options } = this.props;
+        const currentOptionIndex = options.findIndex(option => isOptionChecked(option, value));
+        return currentOptionIndex !== -1 ? this.optionsRefs[currentOptionIndex] : null;
+    }
+
+    generateRefsForOptions() {
+        const { options } = this.props;
+        return options.map(() => React.createRef());
+    }
+
+    addRefsToOptions(options) {
+        return options.map((option, index) => ({
+            optionRef: this.optionsRefs[index],
+            ...option,
```

Comment:
this need to be in this order:
```
...option,
optionRef: this.optionsRefs[index],
```
if you put optionRef first, then the consumer can pass an optionRef ref prop that will cancel the one we want inside and then will break the component